### ISSUE TITLE
Clarification regarding number of epochs for training and choosing a model

### ISSUE THREAD

**Issue by Adithya1976** [hanruihua/rl_rvo_nav#28]
As indicated by this [https://github.com/hanruihua/rl_rvo_nav/issues/6](issue) I noticed that you have trained your model upto 1000+ epochs. However, the arguments in train_process_s1 and train_process_s2 indicate 250 epochs each. To obtain your final policy, can you please let me know how many epochs you trained in each phase (4 robots and 10 robots respectively). After training all your models, how did you decide on which checkpoint to use as your final policy? Did you choose that by post-train results?
Additionally, please confirm if all of the other parameters such as training and reward parameters are exactly as indicated in the research paper and the training code. Please do let me know if youve used different parameters for training.

**Comment by hanruihua** [comment]
Hi,

The epoch arguments in `train_process_s1.py` and `train_process_s2.py` are set to 250 and 2000, respectively. This means the policy is initially trained with 4 robots for 250 epochs and then continues training with 10 robots for 2000 epochs. In my experience, the policy performance may converge between 1200 and 1500 epochs.

During the training process, another process regularly tests and prints the trained policy’s performance. You can select the checkpoint with the best performance based on these results.

We provide the pretrained model along with the training and reward parameters, which can be found at https://github.com/hanruihua/rl_rvo_nav/tree/main/rl_rvo_nav/pre_trained_model/pre_trained. You can refer to these parameters for your training. Please note that the performance is sensitive to the reward parameters, which might need slight adjustments in different environments.
